### DISCUSSION TITLE
Modular architecture

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#12327] [Help]
### Package

Other

### Package Version

v3

### How can we help you?

I am building a modular architecture, could you advise how to generate resources in a different namespace and also register them in the admin panel?

**Comment by leandrocfe** [comment]
you can tell the panel where resource classes should be located

```php
$panel
...
->discoverResources(in: app_path('Filament/Resources'), for: 'App\\Filament\\Resources')
```

**Reply by yablochnyi** [reply]
You misunderstood me. I mean that I want to generate resources with the specification of the module directory, that is, using 
```
php artisan make:filament-resource User --specify the directory where to place it 
```
Something like that.
